tests/test_extract_lyrics.py

```diff
@@ -0,0 +1,65 @@
+from pylyrics import extract_lyrics as pl
+import json, os
+import pytest
+
+# add token file to "/data/credentials.json"
+# {
+#   "token": "YOUR-TOKEN"
+# }
+
+
+def get_token():
+    current_path = os.getcwd()
+    with open(current_path + "/tests/data/credentials.json") as f:
+        login = json.load(f)
+    return login["token"]
+
+
+# Case 1 - happy case
+def test_happy_case():
```

Comment:
Recommending on bringing function comments inside:
```suggestion
def test_happy_case():
"""Test a happy case"""
```

Reply:
same recommendation for other functions

Reply:
I used try-except to catch my error message, to make sure it throws the correct error.
Since it is a happy case, it should not throw error, so not useful in this function
[Tiff's test](https://github.com/ttimbers/convertempPy/blob/master/tests/test_convertempPy.py) doesn't use try-except either so I think it's fine